docs/advanced_examples/LoraMLP.ipynb

```diff
@@ -276,224 +267,47 @@
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LoRA layers detected in the model.\n"
+     ]
+    }
+   ],
    "source": [
-    "# Set up LoRA training\n",
-    "lora_training = LoraTraining(peft_model)\n",
-    "\n",
-    "# Set up optimizer and scheduler\n",
+    "# Update training parameters, including loss function\n",
     "optimizer = optim.Adam(filter(lambda p: p.requires_grad, peft_model.parameters()), lr=0.01)\n",
+    "loss_fn = nn.CrossEntropyLoss()\n",
+    "training_args = {\"gradient_accumulation_steps\": 1}\n",
     "\n",
-    "# Update training parameters, including loss function\n",
-    "lora_training.update_training_parameters(\n",
-    "    optimizer=optimizer,\n",
-    "    loss_fn=nn.CrossEntropyLoss(),\n",
-    "    training_args={\"gradient_accumulation_steps\": 1},\n",
+    "# Set up LoRA training\n",
+    "lora_trainer = LoraTrainer(\n",
```

Comment:
Maybe add a comment here, to say that the LoraTrainer uses the hybrid approach.

At this point, everything is perfectly encapsulated, the user doesn't see a hybrid model, but the title does.
So maybe add a comment here.

Reply:
We should probably not mention the hybrid model at this point. It introduces complex topics. I don't think there is any mention to hybrid model?

Reply:
The title mentioned the hybrid model:  `Setup FHE fine-tuning with LoraTraining and HybridFHEModel`

Reply:
Ah yes that's a miss. I will update to mention LoraTrainer instead.